interpreter/astwalker.go

```diff
@@ -38,15 +38,17 @@ const (
 func WalkExpr(expression *exprpb.Expr,
 	metadata Metadata,
 	dispatcher Dispatcher,
-	state MutableEvalState) []Instruction {
+	state MutableEvalState,
+	forceEval bool) []Instruction {
```

Comment:
Consider renaming from `forceEval` to `shortcircuit`.  Removing the `jump` instructions is all that's needed to evaluate all branches.

Reply:
Done~